### ISSUE TITLE
Run Handesk with existing Laravel project

### ISSUE THREAD

**Issue by claudsonm** [BadChoice/handesk#101]
It is possible to run the Handesk with a existing Laravel project? As if it was a common package.

I'm asking this because the installation process aims a clean installation.

**Comment by BadChoice** [comment]
Actually it is meant to run as standalone app, so you don't merge two different things,
But I find that a really easy way to go is to have it on the same server with virtual hosts so you can have

handesk.yourdomain.com and yourdomain.com at the same server

if you need comunication between both apps, you can use the api (https://github.com/BadChoice/handesk-php)

**Comment by brianlmerritt-kent** [comment]
Do users then have to have 2 registrations/logins?  One to the main site and one to the Handesk?

**Comment by BadChoice** [comment]
Right now, as handesk goes appart, they need to, but we can try to work on
something to make it use just one, we can try to create some kind of auth
driver so you can configure it to use your main site’s users

Jordi Puigdellívol

El 5 de marzo de 2018 a las 11:15:42, Brian Merritt (
notifications@github.com) escribió:

Do users then have to have 2 registrations/logins? One to the main site and
one to the Handesk?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/BadChoice/handesk/issues/101#issuecomment-370372431>,
or mute the thread

**Comment by claudsonm** [comment]
It makes sense to run it as a separated thing. But in fact, this thing about having two login systems is not optimal.

But if we had some kind of integration, would be a blast!

**Comment by BadChoice** [comment]
yeah, we would need to have a laravelApp auth driver o something like this, but the problem is that we need to auth it there, so the other app needs some kind of endpoint, or handesk needs to know about the other app key an auth database connection

**Comment by ediblemanager** [comment]
If running on the same server, you could have a background process that adds the users from one site to handesk (assuming both use the same password encryption algorithms). 

Alternatively, they could use the same login details for both - less secure, but easier for users.
